### PR TITLE
deps: update to @bpmn-io/properties-panel@0.20.1

### PR DESCRIPTION
**Pull request by marstamm**
and prepare patch release for updated peer dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,7 +110,7 @@
     "zeebe-bpmn-moddle": "^0.15.0"
   },
   "peerDependencies": {
-    "@bpmn-io/properties-panel": "^0.20.0",
+    "@bpmn-io/properties-panel": "^0.20.1",
```
Comment (pinussilvestrus) [inline package.json:113]: Do we need to be that strict, can't we simply specify `20.x`?

Comment (barmac) [reply package.json:113]: `^0.20.1` is in fact 0.x but not lower than 0.20.1, cf. https://docs.npmjs.com/about-semantic-versioning

Comment (marstamm) [reply package.json:113]: That's true, but not consistent with how we handle the others. I missed it as well

**Comment by marstamm** [comment]
Closing in favour of https://github.com/bpmn-io/bpmn-js-properties-panel/pull/754, as we don't need the release for peer dependencies

**Pull request closed without merging**